### DISCUSSION TITLE
[Feature] Intellisense or Typeahead When Changing Person Name From Explore People Page

### DISCUSSION THREAD

**Discussion by kramttocs** [immich-app/immich#13900] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When on the Explore > People page if you use the 3 dots > Change Name it doesn't prefill/typeahead with names already in the system

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5206
